builds/misc/templates/move-rust-artifacts.yaml

```diff
@@ -0,0 +1,41 @@
+# This template will move the executables from a rust target folder into a destination folder.
+# It will move only the executables while preserving the structure of the target folder.
+# The destination folder has the following structure:
+#
+# [destination.folder.path]/
+#  - [target.amd64]/release
+#     - [artifact.name]
+#  - [target.arm32v7]/release
+#     - [artifact.name]
+#  - [target.arm64v8]/release
+#     - [artifact.name]
+parameters:
+  target.arm32v7: 'armv7-unknown-linux-gnueabihf'
```

Comment:
nit: can params be reduced to 
- target: 'armv7-unknown-linux-gnueabihf' | ....
- arch: 'arm32v7' | 'arm64v8' | 'amd64'

Reply:
I wanted to avoid having to specify the target when calling the template. If you still think this way is better then I can change it.